unstructured_inference/inference/elements.py

```diff
@@ -216,6 +192,20 @@ def extract_text(
             text = ""
         return text
 
+    @classmethod
+    def from_coords(
+        cls,
+        x1: Union[int, float],
+        y1: Union[int, float],
+        x2: Union[int, float],
+        y2: Union[int, float],
+        **kwargs,
```

Comment:
can we have the optional variables explicitly listed here please so that it can be helpful to devs using hints.

Reply:
The purpose here is to support subclassing without having to override the classmethod whenever the constructor changes. If I add the optional kwargs from this class and keep the `**kwargs` for potential subclasses, would you feel good about that?

Reply:
Made the above changes in [844868b](https://github.com/Unstructured-IO/unstructured-inference/pull/232/commits/844868b287f9b8360aaaad7da028a5d99028c057). LMK if this works.

Reply:
maybe this is not the PR to discuss this fine print detail; this can be a follow up discussion on explicit vs. implicit patterns. The `kwarg` here make this class ambiguous: it is not clear from definition what parameters is can accept. If a subclass has need for more parameters for this function they obviously would just override this method so we need a newly defined function anyway. And we'd only pass those new parameters when using the subclass that is supposed to support. But this shouldn't block this PR I guess.

Reply:
Overriding the method isn't needed this way, that's the advantage of doing it this way (as evidenced by the fact that mostly it's the `LayoutElement` subclass of `TextRegion` that uses this method without overriding it)... but granted maybe it's not worth the ambiguity.